artemis-server/src/main/java/org/apache/activemq/artemis/core/protocol/ServerPacketDecoder.java

```diff
@@ -115,6 +123,8 @@ public Packet decode(final ActiveMQBuffer in) {
       switch (packetType) {
          case SESS_SEND:
             return decodeSessionSendMessage(in);
+         case SESS_SEND_V2:
+            return decodeSessionSendMessageV2(in);
```

Comment:
If decodeSessionSendMessageV2 is not supposed to be the common case here, would be better to move it on the `slowPathDecode`  instead

Reply:
it is common case, it will replace SESS_SEND

Reply:
This will clash with my changes...  that's about to be merged today.

Reply:
Thats fine, any clashes i will sort when i rebase, I'm kinda holding off doing any further work right now until you're merges tbh.

Reply:
I have rebased and sorted, was easy to fix, but thanks for the warning @clebertsuconic